pkg/api/ldap_debug.go

```diff
@@ -141,6 +143,71 @@ func (server *HTTPServer) GetLDAPStatus(c *models.ReqContext) Response {
 	return JSON(http.StatusOK, serverDTOs)
 }
 
+func (server *HTTPServer) PostSyncUserWithLDAP(c *models.ReqContext) Response {
```

Comment:
I would try to split it to smaller functions, it seems cyclomatic complexity is high with this one. Also can use the comment. Yeah, others in this module do not have it, but it seems we can improve that trend :).

--

It seems there is many commonalities between this logic and `ldasync.Sync` method right?